apps/ui/src/views/Space/Editor.vue

```diff
@@ -466,16 +473,27 @@ watchEffect(() => {
             type="error"
             class="mb-4"
           >
-            <span
-              >You can publish up to {{ MAX_1D_PROPOSALS.verified }} proposals
-              per day and {{ MAX_30D_PROPOSALS.verified }} proposals per month.
+            <span>
+              You can publish up to
+              {{
+                Number(
+                  settings.get(`space.verified.proposal_limit_per_day`) || 0
+                )
```

Comment:
It is better to handle `Number` and default values (`0`) in `useSettings` itself than handling them separately

Reply:
I agree, but not sure about the best way to do this. Maybe we should recreate the settings object in `useSettings` using known properties and already fill in defaults/handle conversions there.

Reply:
I added a whitelist of settings inside the composable

The composable will not handle the typecasting, as well as filter out unknown settings

Reply:
Now, typecheck start complaining, since the settings value can either be a string[] or number.

We either add another type casting when using `as Number`, or instead of exposing `settings` from the composable, we store the settings in different Maps, each for each types.


```ts
// in composable, return { limits, list }

// In component, use as:
const { limits } = useSettings();

const limit: Number = limits.get('space.turbo.strategies_limit')
const spaces: string[] = list.get('space.ecosystem.list')
```

Reply:
I think storing them in different variables should be good, one for limits and other for lists